src/Lorentz/Contracts/BaseDAO/Types.hs

```diff
@@ -106,7 +106,11 @@ instance HasAnnotation FA2.Parameter
 
 data Config contractExtra proposalMetadata otherParam = Config
   { cDaoName :: Text
+  -- ^ Name of the DAO.
+  -- Only used in documentation generater and does not get stored in `Storage`.
   , cDaoDescription :: Markdown
+  -- ^ Description of the DAO.
+  -- Only used in documentation generater and does not get stored in `Storage`.
```

Comment:
1. `s/generater/generator` everywhere.
2. In Haddock backticks should be replaced with single quotes or `@`s.